src/test/java/com/wojteknier03/clinicmedical/service/AppointmentServiceTest.java

```diff
@@ -128,6 +128,58 @@ private Appointment createAppointment(Long id) {
         return appointment;
     }
 
+    @Test
+    void assignPatientToAppointment_ValidIds_ExceptionThrown(){
```

Comment:
Nie rozumiem nazwy - dlaczego `ExceptionThrown`? Z testu wynika że jest to happy path